internal/input/clouditor_header.txt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Fraunhofer AISEC
```

Comment:
should we name this .proto?

Reply:
I wouldn't actually, as it is not a proto file. Maybe clouditor_proto_header.txt?

Reply:
Changed to clouditor_header.proto.